src/main/kotlin/no/nav/familie/ks/sak/kjerne/beregning/domene/AndelTilkjentYtelse.kt

```diff
@@ -0,0 +1,142 @@
+package no.nav.familie.ks.sak.kjerne.beregning.domene
+
+import no.nav.familie.ks.sak.common.entitet.BaseEntitet
+import no.nav.familie.ks.sak.common.util.MånedPeriode
+import no.nav.familie.ks.sak.common.util.YearMonthConverter
+import no.nav.familie.ks.sak.kjerne.personident.Aktør
+import java.math.BigDecimal
+import java.time.YearMonth
+import java.util.Objects
+import javax.persistence.CascadeType
+import javax.persistence.Column
+import javax.persistence.Convert
+import javax.persistence.Entity
+import javax.persistence.EnumType
+import javax.persistence.Enumerated
+import javax.persistence.GeneratedValue
+import javax.persistence.GenerationType
+import javax.persistence.Id
+import javax.persistence.JoinColumn
+import javax.persistence.ManyToOne
+import javax.persistence.OneToOne
+import javax.persistence.SequenceGenerator
+import javax.persistence.Table
+
+@Entity(name = "AndelTilkjentYtelse")
+@Table(name = "ANDEL_TILKJENT_YTELSE")
+data class AndelTilkjentYtelse(
```

Comment:
Er dette en god kanditat for å prøve ut [immutable](https://docs.jboss.org/hibernate/orm/5.2/javadocs/org/hibernate/annotations/Immutable.html) hibernateentiteter siden vi nettopp har hatt en kritisk bug i BA-sak der databaseraden ble endret fordi vi muterte på en aty langt nede i koden?